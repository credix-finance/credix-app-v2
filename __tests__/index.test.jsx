import renderer from "react-test-renderer";
import Overview from "../src/pages/index";

describe("Home", () => {
	it("renders a heading", () => {
		const component = renderer.create(<Overview />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
