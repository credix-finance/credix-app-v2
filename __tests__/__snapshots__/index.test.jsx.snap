// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a heading 1`] = `
<main
  className="grid grid-cols-1 grid-auto-rows-min gap-y-8 md:grid-cols-12 md:gap-y-12 md:gap-x-14 justify-items-center p-4 pt-8 md:p-8 lg:pt-16 lg:max-w-6xl lg:justify-self-center"
  id="index"
>
  <div
    className="text-center md:col-span-12 md:max-w-3xl grid justify-items-center"
  >
    <h1
      className="text-5xl md:text-6xl lg:text-7xl font-semibold font-sans"
    >
      Welcome to Credix
    </h1>
    <div
      className="font-normal text-base md:max-w-lg"
    >
      The new decentralized credit marketplace connecting investors with FinTechs in emerging markets
    </div>
  </div>
  <div
    className="md:col-span-4 w-full flex justify-center"
  >
    <div
      className="bg-credix-primary border border-solid border-darker rounded-[1px] font-sans h-36 w-min min-w-[16rem] md:w-full md:min-w-[12rem] ml-[21.5px] pr-5 flex items-center"
    >
      <div
        className="bg-credix-primary ml-[-21.5px]"
      >
        <div>
          TVL
        </div>
        <div
          className="text-6xl font-bold"
        >
          14.8M
          <span
            className="text-sm font-normal"
          >
            USDC
          </span>
        </div>
      </div>
    </div>
  </div>
  <div
    className="md:col-span-4 w-full flex justify-center"
  >
    <div
      className="bg-credix-primary border border-solid border-darker rounded-[1px] font-sans h-36 w-min min-w-[16rem] md:w-full md:min-w-[12rem] ml-[21.5px] pr-5 flex items-center"
    >
      <div
        className="bg-credix-primary ml-[-21.5px]"
      >
        <div>
          Estimatded APY
        </div>
        <div
          className="text-6xl font-bold"
        >
          13.8%
          <span
            className="text-sm font-normal"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="md:col-span-4 w-full flex justify-center"
  >
    <div
      className="bg-credix-primary border border-solid border-darker rounded-[1px] font-sans h-36 w-min min-w-[16rem] md:w-full md:min-w-[12rem] ml-[21.5px] pr-5 flex items-center"
    >
      <div
        className="bg-credix-primary ml-[-21.5px]"
      >
        <div>
          Credit outstanding
        </div>
        <div
          className="text-6xl font-bold"
        >
          12.3M
          <span
            className="text-sm font-normal"
          >
            USDC
          </span>
        </div>
      </div>
    </div>
  </div>
  <div
    className="ml-6 md:col-span-12 md:flex md:justify-between md:space-x-20 space-y-8 md:space-y-0"
  >
    <div
      className="md:col-span-3 border border-solid rounded-[1px] border-darker w-full p-7 pt-16"
    >
      <div
        className="bg-credix-primary -ml-14 py-[18.5px]"
      >
        <div
          className="text-xs font-normal capitalize"
        >
          liquidity providers
        </div>
        <div
          className="font-sans font-semibold text-[32px] capitalize"
        >
          invest
        </div>
      </div>
      <div
        className="text-base"
      >
        <div
          className="mb-14 text-base"
        >
          Stable return, flexibility to withdraw at any moment and invest in senior tranche = liquidity pool.
        </div>
        <button
          className="ant-btn ant-btn-primary ant-btn-block rounded-[1px] border text-shadow-none shadow-none flex items-center justify-center gap-2 px-[25px] 
				bg-action-primary border-action-primary text-credix-primary
				hover:bg-neutral-60 hover:border-neutral-60
				active:bg-neutral-40 active:border-neutral-40 active:text-neutral-100
				disabled:bg-action-disable disabled:border-transparent disabled:text-disabled
			 text-base font-medium h-[42px] capitalize"
          onClick={[Function]}
          type="button"
        >
          <span>
            Invest
          </span>
        </button>
      </div>
    </div>
    <div
      className="md:col-span-3 border border-solid rounded-[1px] border-darker w-full p-7 pt-16"
    >
      <div
        className="bg-credix-primary -ml-14 py-[18.5px]"
      >
        <div
          className="text-xs font-normal capitalize"
        >
          Borrowers
        </div>
        <div
          className="font-sans font-semibold text-[32px] capitalize"
        >
          borrow
        </div>
      </div>
      <div
        className="text-base"
      >
        <div
          className="mb-14 text-base"
        >
          Stable return, flexibility to withdraw at any moment and invest in senior tranche = liquidity pool.
        </div>
        <button
          className="ant-btn ant-btn-primary ant-btn-block rounded-[1px] border text-shadow-none shadow-none flex items-center justify-center gap-2 px-[25px] 
				bg-action-primary border-action-primary text-credix-primary
				hover:bg-neutral-60 hover:border-neutral-60
				active:bg-neutral-40 active:border-neutral-40 active:text-neutral-100
				disabled:bg-action-disable disabled:border-transparent disabled:text-disabled
			 text-base font-medium h-[42px] capitalize"
          onClick={[Function]}
          type="button"
        >
          <span>
            deals
          </span>
        </button>
      </div>
    </div>
  </div>
</main>
`;
